test/puppetlabs/puppetdb/integration/db_resilience/db_fallback.clj

```diff
@@ -0,0 +1,22 @@
+(ns puppetlabs.puppetdb.integration.db-resilience.db-fallback
```

Comment:
I think it's valuable, but we can't actually test that here. It's checking that we keep working when postgres itself goes away and comes back. i.e. we're using the connection pool appropriately. I don't know how to replicate that in this context. If we moved to self-hosting postgres out of the tests, we could do it.